graphqlapi/src/main/resources/config.yaml

```diff
@@ -0,0 +1,7 @@
+server:
+  applicationConnectors:
+    - type: http
+      port: 8080
+  adminConnectors:
+    - type: http
+      port: 8091
```

Comment:
The convention in other bundles is to use applicationPort + 1, but 8081 is already taken by the auth API.

I wish we would simply remove those admin connectors, they serve no purpose because health-checker is supposed to centralize those features. But if we do it introduces a bug because of this line in `Starter.java`:
```java
System.setProperty("dw.server.adminConnectors[0].bindHost", listenHostStr);
```

Reply:
And we can't get rid of that line in Starter.java?

Reply:
No, because it still applies to the health-checker module.